mgmtsystem/azure.py

```diff
@@ -297,6 +297,7 @@ def delete_stack(self, stack_name, resource_group=None):
             -DeploymentName \"{stack}\" -Force
             }}
             """.format(rg=resource_group or self.resource_group, stack=stack_name))
+        self.delete_vm(stack_name, resource_group or self.resource_group)
```

Comment:
We can't do that, that assumes that anyone using this is always going to have a vm_name that matches the stack_name. Whilst we do this in our tests, we can't do this in wrapanapi.